behaviors/README.md

```diff
@@ -197,6 +197,28 @@ footerMetadata:
 
 Like the Page Area property, this only works in a layout's component lists.
 
+#### Fuzzy Component Lists and Properties
+
+By default, component lists and properties allow strict whitelisting and blacklisting with `include` and `exclude`. This is appropriate for most situations, but there are certain scenarios where you might want to be less strict. For example, the body of an article might be paragraphs and images 99% of the time, but you still want to allow your interactives team to create one-off components and add them to articles.
+
+You can accomplish this by adding `fuzzy: true` to your component list / property. This will add a "View all components" button to the bottom of the component list in the add-components pane. When a user clicks the button, it'll open a different add-components pane with a list of all available components.
```

Comment:
No, it makes the white/black list less strict by allowing users to (with an extra action) add _any_ component available in the clay instance. This allows for things like one-off interactives in articles (without having to manually to api calls against the server to add them)